### ISSUE TITLE
Phase 1 overambition

### ISSUE THREAD

**Issue by jeffnv** [antongb/AudioNimbus#3]
Phase one should be at least 2 phases. Getting Uploading, Showing, Playing of tracks working well in backbone views will NOT be trivial

**Comment by rglassett** [comment]
Agreed. Start by showing/playing a couple seed tracks, then do file upload in its own layer.
